docs/chat_setup/mattermost_setup.md

```diff
@@ -25,7 +25,7 @@
    - Copy the displayed url matching the Server you want to add the chatbot to.
      - Make sure to include the `http://` or `https://`.
      - If shown, make sure to include the port. Ex. `https://example.com:8065`
-6. Proceed to the [Nautobot Server Preparation and Configuration](./chat_setup.md#Nautobot Server Preparation and Configuration) section.
+6. Proceed to the [Nautobot Server Preparation and Configuration](./chat_setup.md#Plug-In-Installation) section.
```

Comment:
FYI looks like there's also a broken link at step 4 above - mind fixing it as well?

Reply:
Thanks. I'll see if I can take a deeper dive through the docs again to check for it.